sample/build.gradle

```diff
@@ -72,7 +72,7 @@ dependencies {
     implementation project(':clevertap-core')
 
     implementation fileTree(include: ['*.aar'], dir: 'libs')
-    //implementation "com.clevertap.android:clevertap-android-sdk:4.7.3"
+    //implementation "com.clevertap.android:clevertap-android-sdk:4.7.4"
```

Comment:
@piyush-kukadiya versionName "1.4.6-push-plus" -- in sample build.gradle can you update the version name to the latest one